modules/webxr/doc_classes/WebXRInterface.xml

```diff
@@ -73,7 +74,10 @@
 		    # This will be the reference space type you ultimately got, out of the
 		    # types that you requested above. This is useful if you want the game to
 		    # work a little differently in 'bounded-floor' versus 'local-floor'.
-		    print ("Reference space type: " + webxr_interface.reference_space_type)
+		    print ("Reference space type: ", webxr_interface.reference_space_type)
+		    # This will be the list of features that were successfully enabled
+		    # (except on browsers that don't support this property).
+		    print ("Enabled features: ", webxr_interface.enabled_features)
```

Comment:
```suggestion
		    print("Reference space type: ", webxr_interface.reference_space_type)
		    # This will be the list of features that were successfully enabled
		    # (except on browsers that don't support this property).
		    print("Enabled features: ", webxr_interface.enabled_features)
```

Reply:
Ah, yes, a good time to fix that stylistic oddity :-)

Fixed in my latest push!